src/Integration.Vsix.UnitTests/Settings/SonarLintSettingsTests.cs

```diff
@@ -22,354 +22,287 @@
 using FluentAssertions;
 using Microsoft.VisualStudio.Settings;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
 using SonarLint.VisualStudio.Integration.Vsix;
+using SonarLint.VisualStudio.Integration.Vsix.Settings;
 using SonarLint.VisualStudio.TestInfrastructure;
 
 namespace SonarLint.VisualStudio.Integration.UnitTests.Settings
 {
     [TestClass]
     public class SonarLintSettingsTests
     {
-        private ConfigurableWritableSettingsStore settingsStore;
```

Comment:
I re-wrote the existing tests to get rid of the `TestInitialize`, and to change the tests to be data-driven. I suggest ignoring the old tests; I'll describe the behaviour of the new tests.